app/src/main/AndroidManifest.xml

```diff
@@ -12,8 +12,6 @@
 
     <application
         android:networkSecurityConfig="@xml/network_security_config"
-        android:icon="@mipmap/ic_launcher_foreground"
-        android:roundIcon="@mipmap/ic_launcher_round"
```

Comment:
Hvorfor slette disse?

Reply:
Ser ut som en merge error? He ikkje endra på dei filene, men ser dei he blitt flytta fra mipmap til drawable

Reply:
hmm.... hvis jeg ikke tar feil så vil iconet forsvinne hvis ikke disse to linjene er med